assets/src/settings-page/paired-url-structure.js

```diff
@@ -88,6 +88,173 @@ function getCustomPairedStructureSources( sources ) {
 	return message;
 }
 
+/**
+ * Slug conflicts notice.
+ *
+ * @param {Object} props           Component props.
+ * @param {string} props.slug      Slug.
+ * @param {Object} props.conflicts Conflicts.
+ */
+function SlugConflictsNotice( { slug, conflicts } ) {
+	return (
+		<AMPNotice size={ NOTICE_SIZE_LARGE } type={ NOTICE_TYPE_INFO }>
+			<p>
+				{
+					sprintf(
+						/* translators: %s is the AMP slug */
+						__( 'There are one or more entities that are already using the “%s” URL slug. For this reason, you cannot currently use the path suffix or legacy reader paired URL structures. See below for the source of the slug conflict(s):', 'amp' ),
+						slug,
+					)
+				}
+			</p>
+			<ul>
+				{ conflicts.posts && (
+					conflicts.posts.map( ( entity ) => (
+						<li key={ `post-${ entity.id }` }>
+							{
+								entity.edit_link ? (
+									<a href={ entity.edit_link } target="_blank" rel="noreferrer">
+										{ entity.label || entity.post_type }
+									</a>
+								) : (
+									entity.label || entity.post_type
+								)
+							}
+							{
+								entity.title && (
+									': ' + entity.title
+								)
+							}
+							{ ' ' }
+							<small>
+								{
+									/* translators: %d is entity ID */
+									sprintf( __( '(ID: %d)', 'amp' ), entity.id )
+								}
+							</small>
+						</li>
+					) )
+				) }
+
+				{ conflicts.terms && (
+					conflicts.terms.map( ( entity ) => (
+						<li key={ `term-${ entity.id }` }>
+							{
+								entity.edit_link ? (
+									<a href={ entity.edit_link } target="_blank" rel="noreferrer">
+										{ entity.label || entity.post_type }
+									</a>
+								) : (
+									entity.label || entity.post_type
+								)
+							}
+							{
+								entity.name && (
+									': ' + entity.name
+								)
+							}
+							{ ' ' }
+							<small>
+								{
+									/* translators: %d is entity ID */
+									sprintf( __( '(ID: %d)', 'amp' ), entity.id )
+								}
+							</small>
+						</li>
+					) )
+				) }
+
+				{
+					conflicts.user && (
+						[ conflicts.user ].map( ( entity ) => (
```

Comment:
Any reason for wrapping this in an array and then `map`ping it?

Reply:
(See suggestion below)

Reply:
Mainly just to make the style similar to the preceding two, but it's not needed.